todo_app/app.py

```diff
@@ -1,41 +1,116 @@
+import functools
 from flask import Flask, render_template, request, redirect
-from todo_app.todo import Item
 from todo_app.flask_config import Config
 from todo_app.view_model import ViewModel
 from todo_app.todomongo import add_todo_item, items, update_status, delete_item
+import requests
+import os
+from flask_login import LoginManager, UserMixin, login_required, login_user, current_user
+
+class User (UserMixin):
+    def __init__(self, id):
+        self.id = id
+        self.is_reader = True
+        if id == '97612224':
```

Comment:
Can confirm this worked for me as expected - when this wasn't my ID I couldn't see the buttons (and even if I cheated, I couldn't bypass your security!) and when it was I could use the app as if I had full permissions 😄